data_steward/cdr_cleaner/cleaning_rules/map_health_insurance_responses.py

```diff
@@ -193,82 +177,141 @@ def setup_rule(self, client, *args, **keyword_args):
+                SANDBOXED_INSURANCE_ROWS),
             pipeline_tables=PIPELINE_TABLES,
-            pids_lookup_table=PIDS_LOOKUP_TABLE)
+            health_insurance_pids=HEALTH_INSURANCE_PIDS)
         queries.append(sandbox_query)
 
         invalidate_query = dict()
         invalidate_query[cdr_consts.QUERY] = UPDATE_INVALID_QUERY.render(
             project_id=self.project_id,
-            combined_dataset_id=self.dataset_id,
+            dataset_id=self.dataset_id,
             ORIGINAL_OBSERVATION_SOURCE_CONCEPT_ID=
             ORIGINAL_OBSERVATION_SOURCE_CONCEPT_ID,
             pipeline_tables=PIPELINE_TABLES,
-            pids_lookup_table=PIDS_LOOKUP_TABLE)
+            health_insurance_pids=HEALTH_INSURANCE_PIDS)
         queries.append(invalidate_query)
 
-        delete_query = dict()
-        delete_query[
-            cdr_consts.QUERY] = DELETE_ORIGINAL_FOR_HCAU_PARTICIPANTS.render(
+        new_insurance_rows_query = dict()
+        new_insurance_rows_query[
+            cdr_consts.QUERY] = CREATE_NEW_INSURANCE_ROWS.render(
                 project_id=self.project_id,
-                combined_dataset_id=self.dataset_id,
+                dataset_id=self.dataset_id,
                 sandbox_dataset_id=self.sandbox_dataset_id,
-                new_insurance_rows=NEW_INSURANCE_ROWS,
+                new_insurance_rows=self.sandbox_table_for(NEW_INSURANCE_ROWS),
+                insurance_lookup=INSURANCE_LOOKUP,
                 ORIGINAL_OBSERVATION_SOURCE_CONCEPT_ID=
-                ORIGINAL_OBSERVATION_SOURCE_CONCEPT_ID)
-        queries.append(delete_query)
+                ORIGINAL_OBSERVATION_SOURCE_CONCEPT_ID,
+                HCAU_OBSERVATION_SOURCE_CONCEPT_ID=
+                HCAU_OBSERVATION_SOURCE_CONCEPT_ID,
+                pipeline_tables=PIPELINE_TABLES,
+                health_insurance_pids=HEALTH_INSURANCE_PIDS)
+        queries.append(new_insurance_rows_query)
 
-        insert_query = dict()
-        insert_query[
-            cdr_consts.QUERY] = INSERT_ANSWERS_FOR_HCAU_PARTICIPANTS.render(
-                project_id=self.project_id,
-                combined_dataset_id=self.dataset_id,
-                sandbox_dataset_id=self.sandbox_dataset_id,
-                new_insurance_rows=NEW_INSURANCE_ROWS)
-        queries.append(insert_query)
+        update_basics_query = dict()
+        update_basics_query[cdr_consts.QUERY] = UPDATE_HCAU_ROWS.render(
+            project_id=self.project_id,
+            dataset_id=self.dataset_id,
+            ORIGINAL_OBSERVATION_SOURCE_CONCEPT_ID=
+            ORIGINAL_OBSERVATION_SOURCE_CONCEPT_ID,
+            HCAU_OBSERVATION_SOURCE_CONCEPT_ID=
+            HCAU_OBSERVATION_SOURCE_CONCEPT_ID,
+            new_insurance_rows=self.sandbox_table_for(NEW_INSURANCE_ROWS),
+            sandbox_dataset_id=self.sandbox_dataset_id,
+            pipeline_tables=PIPELINE_TABLES,
+            health_insurance_pids=HEALTH_INSURANCE_PIDS)
```

Comment:
`ORIGINAL_OBSERVATION_SOURCE_CONCEPT_ID`, `HCAU_OBSERVATION_SOURCE_CONCEPT_ID`, `pipeline_tables`, and `health_insurance_pids`.

These are arguments that are not asked for in the query and should not be provided. 

```suggestion
                update_basics_query[cdr_consts.QUERY] = UPDATE_HCAU_ROWS.render(
            project_id=self.project_id,
            dataset_id=self.dataset_id,
            sandbox_dataset_id=self.sandbox_dataset_id,
            new_insurance_rows=self.sandbox_table_for(NEW_INSURANCE_ROWS))
```